aspen/http/resource.py

```diff
@@ -0,0 +1,75 @@
+from .response import Response
+from ..simplates import Simplate, SimplateDefaults, SimplateException
+
+
+class Static(object):
+    """Model a static HTTP resource.
```

Comment:
It means to bring [web resources](https://en.wikipedia.org/wiki/Web_resource) under the Aspen [object model](https://en.wikipedia.org/wiki/Object_model). An object model is:

> A collection of objects or classes through which a program can examine and manipulate some specific parts of its world. In other words, the object-oriented interface to some service or system. Such an interface is said to be the object model of the represented service or system.